package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "0.1.35",
+  "version": "0.1.36",
```

Comment:
should this be a minor update?

Reply:
yeah but I just merged something in so you might want to bump again :)

Reply:
yeah but I just merged something in so you might want to bump again :)